sdk/eventgrid/azure-eventgrid/CHANGELOG.md

```diff
@@ -1,14 +1,10 @@
 # Release History
 
-## 4.13.1 (Unreleased)
+## 4.14.0 (2023-09-12)
 
 ### Features Added
 
-### Breaking Changes
-
-### Bugs Fixed
-
-### Other Changes
+- Added new enum values to `SystemEventNames` related to Azure App Configuration and Azure EventGrid.
```

Comment:
maybe list out the new values that were added?

Reply:
I'm following the other change log entries which list the service names instead of the system events by name!